sources/source-Send_to_MQTT.module

```diff
@@ -0,0 +1,89 @@
+<?php
+/*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
+ * Developer Notes:
+ *
+ * Version History
+ *     2020-10-14  Initial creation by hartmann1970@gmx.de
+ *     2020-10-15 added user, password, clientID by hartmann1970@gmx.de
+ *
+ *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***/
+
+class Send_to_MQTT extends superfecta_base {
+
+    public $description = "This source will send CID number and name to an MQTT topic for external processing. The MQTT topic is added with DID and payload is JSON format. To use this feature, mosqitto has to be installed.";
```

Comment:
mosquitto is misspelled in the description. This is an easy note to miss, and mosquitto is not installed by default in the FreePBX Distro. Would be good to test for it being installed and DebugPrint a warning that it's required.

Reply:
Thanks. Missed that typo...